src/Scene.cs

```diff
@@ -231,7 +234,8 @@ private Cell[] CalculateRawCells(List<Cell> groundCells, List<Cell> gameFieldCel
 
                 for (int j = 0; j < frame.Length; j++)
                 {
-                    if (frame[j].X == cell.X && frame[j].Y == cell.Y)
+                    //if (frame[j].X == cell.X && frame[j].Y == cell.Y)
```

Comment:
If that's unnecessary, why to keep this at all? Commented code makes code uglier, isn't it?